api/v1beta1/topology.go

```diff
@@ -34,3 +37,21 @@ func LinkKuadrantToGatewayClasses(objs controller.Store) machinery.LinkFunc {
 		},
 	}
 }
+
+func LinkKuadrantToAuthorino(objs controller.Store) machinery.LinkFunc {
+	kuadrants := lo.Map(objs.FilterByGroupKind(KuadrantKind), controller.ObjectAs[*Kuadrant])
+
+	return machinery.LinkFunc{
+		From: KuadrantKind,
+		To:   schema.GroupKind{Group: authorinov1beta1.GroupVersion.Group, Kind: "Authorino"},
+		Func: func(child machinery.Object) []machinery.Object {
+			parents := make([]machinery.Object, len(kuadrants))
+			for _, parent := range kuadrants {
+				if parent.Namespace == child.GetNamespace() {
+					parents = append(parents, parent)
+				}
+			}
+			return parents
```

Comment:
Nothing wrong with the current approach, but just a minor suggestion leveraging `github.com/samber/lo`:

```suggestion
			return lo.Filter(kuadrants, func(kuadrant machinery.Object, _ int) {
				kuadrant.Namespace == child.GetNamespace()
			})
```